lib/ts/recipe/thirdparty/components/themes/styles.ts

```diff
@@ -40,46 +40,53 @@ export function getStyles(palette: NormalisedPalette): NormalisedDefaultStyles {
 
         providerButton: {
             // Default button height in 34
-            minHeight: "34px",
+            minHeight: "32px",
             // this will allow the button to scale with different font sizes and text lengths
-            height: "auto !important",
+            height: "auto",
             display: "flex",
             flexDirection: "row",
+            alignItems: "center",
             paddingLeft: "0px",
             paddingRight: "0px",
             // This makes the button look somewhat cleaner if the text wraps
             paddingTop: "2px",
             paddingBottom: "2px",
+            "&:hover": {
+                filter: "none !important",
+                backgroundColor: "rgb(250,250,250)",
+            },
         },
 
         providerButtonLeft: {
-            width: "40px",
+            marginRight: "8px",
+            marginLeft: "78px",
         },
 
         providerButtonLogo: {
             height: "30px",
             display: "flex",
-            borderRight: "1px solid rgba(255, 255, 255, 0.6)",
         },
 
         providerButtonLogoCenter: {
+            display: "flex",
             margin: "auto",
         },
 
         providerButtonText: {
-            margin: "auto",
+            fontFamily: "Rubik",
```

Comment:
I don't think we need to set this here

Reply:
do you want me to set it somewhere else?

Reply:
We are using Rubik by default everywhere, doesn't it get applied here?